integration_tests/test_fusion.py

```diff
@@ -0,0 +1,288 @@
+            renderapi.client.import_tilespecs_parallel(
+                child['stack'], ztspecs, tforms,
+                poolsize=pool_size, close_stack=False, render=r)
+            renderapi.client.transformSectionClient(
+                child['stack'], '{}_tform'.format(child['stack']),
+                last_tform.className, last_tform.dataString.replace(' ', ','),
+                zValues=zs, render=r)
+            renderapi.stack.set_stack_state(
+                child['stack'], 'COMPLETE', render=r)
+
+    build_childstacks(
+        nodes[root_index+1:], stackzs[root_index+1:],
+        test_tilespecs, test_tforms, render)
+    build_childstacks(
+        nodes[:root_index][::-1], stackzs[:root_index][::-1],
+        test_tilespecs, test_tforms, render)
+
+    # generate root stack
+    renderapi.stack.create_stack(nodes[root_index]['stack'], render=render)
+    renderapi.client.import_tilespecs_parallel(
+        nodes[root_index]['stack'],
+        [ts for ts in test_tilespecs if ts.z in stackzs[root_index]],
+        test_tforms,
+        poolsize=pool_size, render=render)
+
+    yield dag
+
+    # remove all stacks associated with DAG
+    for nd in nodes:
+        renderapi.stack.delete_stack(nd['stack'], render=render)
+
+
+def get_random_rigid_tform(max_translation_tuple=(10, 10)):
+    # random translation
+    x, y = numpy.array(max_translation_tuple) * numpy.random.rand(2)
+    translate_tform = renderapi.transform.AffineModel(B0=x, B1=y)
+    # random-ish rotation
+    theta = numpy.random.rand() * 2 * numpy.pi
+    rotate_tform = renderapi.transform.AffineModel(
+        M00=numpy.cos(theta), M01=-numpy.sin(theta),
+        M10=numpy.sin(theta), M11=numpy.cos(theta))
+    return translate_tform.concatenate(rotate_tform)
+
+
+def test_register_first_stack(render, stack_DAG, tmpdir):
+    from rendermodules.fusion.register_adjacent_stack import (
+        RegisterSubvolumeModule, example_parameters)
+
+    parentstack = stack_DAG['stack']
+    child = stack_DAG['children'][0]
+    childstack = child['stack']
+    tform = child['transform']
+
+    test_input = dict(example_parameters, **{
+        'render': render.make_kwargs(),
+        'stack_a': parentstack,
+        'stack_b': childstack})
+
+    outputfn = os.path.join(str(tmpdir), 'register_stack_out.json')
+    rms = RegisterSubvolumeModule(
+        input_data=test_input,
+        args=['--output_json', outputfn])
+    rms.run()
+
+    with open(rms.args['output_json'], 'r') as f:
+        outd = json.load(f)
+
+    estimated_tform = renderapi.transform.load_transform_json(
+        outd['transform'])
+    tspecs = renderapi.tilespec.get_tile_specs_from_stack(  # noqa: F841
+        childstack, render=render)
+    assert numpy.allclose(estimated_tform.M, tform.M)
+    assert outd['stack_a'] == parentstack
+    assert outd['stack_b'] == childstack
+
+
+def test_register_all_stacks(render, stack_DAG):
+    from rendermodules.fusion.register_adjacent_stack import (

```

Comment:
@mahaling I think this was an effort to have module imports fail at the level of an individual test rather than the larger test module.  It doesn't match the pattern used in the rest of this repo, so I have changed the test.